gulpfile.babel.js

```diff
@@ -95,6 +95,10 @@ function sass() {
     .pipe(browser.reload({ stream: true }));
 }
 
+function fonts() {
+  return gulp.src('./bower_components/foundation-icon-fonts/**/*.{ttf,woff,woff2,eot,svg}').pipe(gulp.dest('dist/assets/fonts'));
```

Comment:
Instead of hard-coding `./bower_components/foundation-icon-fonts` in the Gulpfile, you can add the path in `config.yml` (as `PATHS.fonts`), but with the `{ttf,woff,woff2,eot,svg}` filter staying in the gulpfile.

Also, don't forget ` src/assets/fonts` ;)